dashboard/src/components/DeploymentFormBody/DeploymentFormBody.tsx

```diff
@@ -131,6 +142,9 @@ class DeploymentFormBody extends React.Component<
           <button className="button button-primary" type="submit">
             Submit
           </button>
+          <button className="button" type="button" onClick={this.openRestoreDefaultValuesModal}>
+            Restore Defaults Values
```

Comment:
Not sure whether "Restore Chart Defaults" would be clearer? Fine either way.

Reply:
Right, that's true